addons/stock/models/stock_quant.py

```diff
@@ -358,13 +361,53 @@ def _compute_complete_name(self):
         return res
 
     def _compute_current_picking_info(self):
+        """ When a package is in displayed in picking, it gets the picking id trough the context, and this function
+        populates the different fields used when we move entire packages in pickings.
+        """
+        for package in self:
+            picking_id = self.env.context.get('picking_id')
+            if picking_id:
+                package.current_picking_move_line_ids = package.move_line_ids.filtered(lambda move_line: move_line.picking_id.id == picking_id)
+                package.current_picking_id = True
+                if package.current_picking_move_line_ids:
```

Comment:
Test is unnecessary. Use `[:1]` instead of `[0]` and it'll give a recordset of size (0, 1), then you can deref an m2o and you'll just get False if the recordset was empty.